reframe/core/decorators.py

```diff
@@ -83,3 +93,32 @@ def _do_register(cls):
         return cls
 
     return _do_register
+
+
+def required_version(*versions):
+    """Class decorator for skipping version-uncompatible tests.
+
+    The decorated class must derive from
+    :class:`reframe.core.pipeline.RegressionTest`. This decorator is also
+    available directly under the :mod:`reframe` module.
+
+    :arg versions: The versions that are compatible with the test.
```

Comment:
Please explain a bit the syntax here. Also you should specify the type of the argument. See other part of the documentation on how this is done.